### DISCUSSION TITLE
Having Dependency Dashboard in separate project

### DISCUSSION THREAD

**Discussion by glensc** [renovatebot/renovate#22128] [Suggest an Idea]
### Type of discussion.

I'm proposing an idea

### Tell us more.

In gitlab we use epics and issue boards, so all issues can be handled in single board.
this means that repositories themselves have issue feature turned off.

is it possible to create Dependency Dashboard issues in some other project?

1. project for issues: /group/issues
2. projects with renovate: /group/repo1, /group/subgroup/repo2

**Comment by viceice** [comment]
no

**Reply by HonkingGoose** [reply]
We closed an old issue for the same feature:

- https://github.com/renovatebot/renovate/issues/7372
